src/test/ui/const-generics/integer-literal-generic-arg-in-where-clause.rs

```diff
@@ -1,7 +1,9 @@
 // check-pass
```

Comment:
Is there a reason why we aren't actually calling these functions here?

It would be nice to change main to
```rust
fn main() {
    takes_closure_of_array_3(returns_closure_of_array_3());
    takes_closure_of_array_3_apit(returns_closure_of_array_3());
}
```

Reply:
I can add that; altho I have no clue why it wasn't that way prior